Frameworks/UIKit/StarboardXaml/LayerTransaction.mm

```diff
@@ -84,7 +86,7 @@
 class QueuedProperty : public ICompositorTransaction {
 public:
     std::shared_ptr<LayerProxy> _layer;
-    char* _propertyName;
+    std::String _propertyName;
```

Comment:
nope! oops!

---
In reply to: [113578228](https://github.com/Microsoft/WinObjC/pull/2636#discussion_r113578228) [](ancestors = 113578228)